styles/components/_accordions.scss

```diff
@@ -11,6 +11,10 @@
   width: 100%;
 }
 
+.sprk-c-Accordion--no-top-border {
```

Comment:
Add a comment above this to explain what it's doing. See the class below for reference. This will also make sure it gets added to the modifiers documentation for Accordion.